internal/bminventory/inventory.go

```diff
@@ -5211,6 +5212,22 @@ func (b *bareMetalInventory) GetInfraEnvByKubeKey(key types.NamespacedName) (*co
 	return infraEnv, nil
 }
 
+func (b *bareMetalInventory) CreateHostInKubeKeyNamespace(ctx context.Context, kubeKey types.NamespacedName, host *models.Host) error {
+	log := logutil.FromContext(ctx, b.log)
+	log.Infof("Register host: %+v (KubeKey: %+v)", host, kubeKey)
```

Comment:
I think the `NamespacedName` type has a `String()` method so you should be able to just print it with `%s`, but also I'd imagine this message is just for debugging since we probably don't want to print the whole host as well?